test/__snapshots__/test.js.snap

```diff
@@ -5248,6 +5402,130 @@ const Impl = require(\\"../implementations/URL.js\\");
 "
 `;
 
+exports[`with processors URLCallback.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+const URL = require(\\"./URL.js\\");
+
+exports.convert = (value, { context = \\"The provided value\\" } = {}) => {
+  if (typeof value !== \\"function\\") {
+    throw new TypeError(context + \\" is not a function\\");
+  }
+
+  function invokeTheCallbackFunction(...args) {
+    const thisArg = utils.tryWrapperForImpl(this);
+    let callResult;
+
+    for (let i = 0; i < Math.min(args.length, 2); i++) {
+      args[i] = utils.tryWrapperForImpl(args[i]);
+    }
+
+    if (args.length < 2) {
+      for (let i = args.length; i < 2; i++) {
+        args[i] = undefined;
+      }
+    } else if (args.length > 2) {
+      args.length = 2;
+    }
+
+    callResult = Reflect.apply(value, thisArg, args);
+
+    callResult = URL.convert(callResult, { context: context });
+
+    return callResult;
```

Comment:
The result should be an impl, since the caller is jsdom code. Right?

Reply:
That’s handled by `URL.convert(…)` above.